### PR TITLE
Fixing zipstrings script to also strings

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N.A

## Description
Added support for non-array strings.
 E.g. if we are passing 2 strings `List1`: "aa" and `List2`:"bb" and `Format`:{a}\{b} , the output will be `aa/bb`

## Screenshots
N.A

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JasBeilin — APPROVED** [review]

File: Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py [hunk 1]
```diff
@@ -8,6 +8,10 @@ def mapper_command(args: Dict[str, Any]) -> CommandResults:
     format = str(args.get('format', '{1}-{2}'))
     list1 = args.get('list1', [])
     list2 = args.get('list2', [])
+    if(isinstance(list1, str )):
```
Comment (JasBeilin) [inline Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py:11]:
```suggestion
    if(isinstance(list1, str)):
```

File: Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py [hunk 2]
```diff
@@ -8,6 +8,10 @@ def mapper_command(args: Dict[str, Any]) -> CommandResults:
...
+        list1=[list1]
+    if(isinstance(list2, str )):
```
Comment (JasBeilin) [inline Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py:13]:
```suggestion
    if isinstance(list2, str):
```

File: Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py [hunk 3]
```diff
@@ -8,6 +8,10 @@ def mapper_command(args: Dict[str, Any]) -> CommandResults:
     format = str(args.get('format', '{1}-{2}'))
     list1 = args.get('list1', [])
     list2 = args.get('list2', [])
+    if(isinstance(list1, str )):
+        list1=[list1]
```
Comment (JasBeilin) [inline Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py:12]:
```suggestion
        list1 = [list1]
```

File: Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py [hunk 4]
```diff
@@ -8,6 +8,10 @@ def mapper_command(args: Dict[str, Any]) -> CommandResults:
...
+    if(isinstance(list2, str )):
+        list2=[list2]
```
Comment (JasBeilin) [inline Packs/CommonScripts/Scripts/ZipStringsArrays/ZipStringsArrays.py:14]:
```suggestion
        list2 = [list2]
```

**Comment by tschanfeld** [comment]
added a period, doc review complete

**Pull request merged**